### DISCUSSION TITLE
帶衰比特幣、災戶遍及美國與台灣：FTX幣圈風暴是什麼？

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#841] [General]
最近因為全球第二大的交易所FTX的事件，讓比特幣等加密貨幣跌的很兇。我覺得這件事跟銀行的擠兌行為很像，一開始有言論說某某銀行快倒了，然後民眾很害怕就會趕快去那家銀行領錢，但是銀行可能沒有那麼多現金，看到銀行付不出錢來就會更緊張，更多人會前來領錢，只是原本以為這件事可以從幣安收購FTX後落幕，結果卻因為FTX遭到美國相關單位調查而破局了，我覺得經過這件事以後，民眾可能會需要一段時間來恢復對加密貨幣的信心。
網址:https://www.google.com/amp/s/www.gvm.com.tw/article/amp/96299

**Comment by manipulateed** [comment]
這個算是加密貨幣的一樁憾事了吧，曾經的輝煌轉瞬即逝，我也一直關注著那一個禮拜的各種新聞，但也很辛運沒有成為受害者，希望受害者都能得到應有的補償

**Comment by Pietra1226** [comment]
感覺也不是虛擬貨幣的問題，該公司的營運一直都有問題，不是一間體制跟制度很全面的公司，太多營運漏洞

**Comment by Chiway1229** [comment]
Ftx的事件確實讓人措手不及，全世界的投資人也至少虧損了十多億美元
此事件不僅告訴了我們在區塊鍊的世界沒有絕對安全的機構、冷錢包是最佳夥伴外
也說明了一個中心化機構在去中心化世界所潛藏的風險

**Comment by peggy0329** [comment]
其實這就是擠兌事件，此事件證明任何投資都有風險，投資人應該要評估自己的風險承擔能力！
而我認為這件事會發生的主要原因還是在FTX這家公司的營運問題上，FTX的母公司Alameda Research有大量負債，而其財務狀況又與FTX綁在一起，而他們又以自家加密貨幣FTT作為抵押品，而當最大投資人幣安大量拋售FTT，使其價格暴跌，那就會造成FTX很嚴重的流動性資金問題，從此事件來看，我認為經營加密貨幣相關產業的業者，不能以大量自身貨幣當作抵押品，以免暴跌後還不出負債，同時加密貨幣是價格風險很高的投資項目，業者不應該過於有效的融資，應該先握有一大筆現金或資產，以作為避險

**Comment by williamyen0623** [comment]
這讓我們意識到爆炸頭的危險（沒
希望加密貨幣市場能越來越好～

**Comment by evanchou1204** [comment]
這件事對幣圈的影響真的很糟，讓人除了幣安之外，哪家公司都不敢相信了QQ

**Comment by eason8479** [comment]
買虛擬幣就是要跳脫機構
結果還有人把幣的所有權交給機構保管

這不就像是想去國外體驗美食
結果人到了國外只敢吃麥當勞

**Comment by yeh-alex** [comment]
所以說交易所應該只是讓人有交易的場所
不應該是能把錢長存的地方
所以錢還是要存在冷錢包內

不過我認為這件事情打擊的只會是對虛擬貨幣不是那麼熟悉的人，畢竟虛擬貨幣去中心化跟儲存、交易方式才是我們目前在虛擬貨幣中追求的目的吧

**Comment by ruthchen1020** [comment]
跟之前的美國經濟大蕭條的故事開頭很像，歷史總是驚人的相似。

**Comment by CPCpyt** [comment]
我想這會對幣圈的監管與信任帶來不少的問題，
光是質押資產和現貨綁定本身就是一個非常離譜的行為，
用錢就質押生成不存在的錢出來，這樣的結構對於知情且有意放空的狙擊手完全是很好的標的。

**Comment by StevenWanglolz** [comment]
SBF那傢伙看起來就賊頭賊腦的，還好我很窮，不會有投資上的問題，因為我沒錢拿去投資哈哈哈 :(

**Comment by wanderer657** [comment]
FTX事件顯示加密貨幣市場脆弱性，缺乏監管使其易受外界影響。與傳統金融體系不同，市場操縱和詐騙活動更容易發生。加強監管和制度建設是重建市場信心的關鍵，需確保市場穩定和透明度，使投資者更有信心參與。

**Comment by asd99263fg** [comment]
虛擬貨幣的風險包括高度波動性帶來的價格劇烈波動，可能導致重大財務損失；交易平台和錢包的安全性問題，可能遭遇黑客攻擊和資金被盜；監管環境的不確定性，可能導致法律和合規風險；詐騙和欺詐行為的風險，由於缺乏監管，容易成為詐騙活動的目標；以及流動性風險，某些虛擬貨幣可能難以轉換為法定貨幣或其他資產。投資者應謹慎評估風險，並考慮多樣化投資和設置風險承受範圍。
